### PR TITLE
move the patients route setting into own function

### PR DESCRIPTION
**Pull request by jh-bate**
@jebeck how does this look? unless I am missing something I think i can also do away with the state.showPatientData also

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -495,80 +495,72 @@ var AppComponent = React.createClass({
...
+      patients: function(cb) {
+        self.fetchInvites(cb);
       }
-
+    },
+    function(err, results) {
+      self.setPatientsRoute();
+      self.renderPage = self.renderPatients;
+      self.setState({page: 'patients'});
+      trackMetric('Viewed Care Team List');
```
Comment (jebeck) [inline app/app.js]: Isn't this metric not necessarily correct since the route can vary?

**Comment by jebeck** [comment]
Yes, I agree about getting rid of `state.showPatientData`. I don't think there's much value in having a boolean in the state that is determined exclusively by *other* parts of the state.

File: app/app.js (hunk 1 above)
Comment (jh-bate) [inline app/app.js]: very true

**Comment by jebeck** [comment]
I just had another look at this because I was looking at this part of the code to figure out my remaining issue with the name + data mismatch. This doesn't help, and I actually think we don't want to pull it in because it simplifies a bit too much - no access to the care team page at all anymore for people who only have access to their own data. Closing for cleanliness sake...

**Pull request closed without merging**